python/modules/csv_managment.py

```diff
@@ -5,6 +5,19 @@
 logger = logging.getLogger("CSV managment")
 logger.info("Executing csv (merge) module.")
 
+def verify_input_file(input_file, delimiter, dimensions = 0):
```

Comment:
@stork119 Dimensions inform us about number of rows and cols in csv file. If the value is different then 0, it means that we wish to verify if given files meet given values requirements. Explanation added.

You're right that name migh be missleading, I'm moving this function to map_plate module and renaming it to verify_input_mapplate_file. 
As far as I know you can't extract the number of rows and colls of file without opening it. At least in python.